packages/media/src/getImageDimensions.js

```diff
@@ -26,7 +26,7 @@ import preloadImage from './preloadImage';
  * @return {Promise} Image dimensions object.
  */
 const getImageDimensions = async (src) => {
-  const img = await preloadImage(src);
+  const img = await preloadImage(src).catch(() => {});
```

Comment:
`preloadImage` is used with `await` here everywhere, would adding `catch` like this do anything?
Probably should also use `try` and `catch` instead.